src/Apps/Settings/Routes/SavedSearchAlerts/SavedSearchAlertsApp.tsx

```diff
@@ -62,7 +62,10 @@ export const SavedSearchAlertsApp: React.FC<SavedSearchAlertsAppProps> = ({
   const [sort, setSort] = useState("ENABLED_AT_DESC")
   const [loading, setLoading] = useState(false)
   const alerts = extractNodes(me.alertsConnection)
-  const isMobile = getENV("IS_MOBILE")
+
+  const xs = __internal__useMatchMedia(THEME.mediaQueries.xs)
```

Comment:
I know that it's not recommended to use it because of server-side rendering, but I do not see any implications of using it here. Maybe @dzucconi or @damassi can say something. In my experience using IS_MOBILE is more harmful

Reply:
It's not server-side friendly (can create very unpredictable and hard to debug problems). You're using it in an effect, so that's fine since it only runs on the client.